README.md

```diff
@@ -1,4 +1,7 @@
 # MySQL Server ROCK
+[![Publish](https://github.com/canonical/charmed-mysql-rock/actions/workflows/publish.yaml/badge.svg)](https://github.com/canonical/charmed-mysql-rock/actions/workflows/publish.yaml)
+[![Tests](https://github.com/canonical/charmed-mysql-rock/actions/workflows/integration.yaml/badge.svg)](https://github.com/canonical/charmed-mysql-rock/actions/workflows/integration.yaml)
```

Comment:
@taurus-forever Agreed - I'd say the names need to sync between all the data platform repos, since there are a lot of inconsistencies there (and the badge names cannot be hardcoded afaik; they will take the `name` field of the workflow or the path). I'll create a ticket for this.